### DISCUSSION TITLE
Custom button generates Alarm state

### DISCUSSION THREAD

**Discussion by LaserNovice77** [arkypita/LaserGRBL#1953] [Q&A]
I have an ORTUR Laser Master 2 S2 LU2-2, 24V Laser Engraver Machine which I purchased in June of this year.  I have used it numerous times to create laser images on wood using LaserGRBL to which I've added various custom buttons, one of which allows me to turn the laser on at low power and frame the project I have loaded so I can see where on my workpiece it will be engraved.  I've used this feature multiple times in the past but this morning when I tried to use that button the engraver stopped responding and went into an alarm state.  Selecting the reset and unlock buttoms allowed me to restart the project but if I again selected the button to turn the laser on at low power and frame the project, it would again go into an alarm state.  I've tried closing and reopening the software, starting a new project, changing settings, etc., but nothing I tried helped.  I don't know if the problem is software or hardware related but the popup error message I get during this process reads:  "LaserGRBL deteced some problem with your Board!  Cause:  MachineAlarm  It would appear that grbl has problems executing this job.  Causes of this situation can be multiple, please read the FAW for information on how to resolve the issue.  https://lasergrbl.comfaw/."  The one FAQ which seems relevant to my problem is "Alarm state - '$H'/'SX' to unlock."  I've followed the suggested procedures with no luck.  I would   appreaciate any help you can provided.

**Comment by arkypita** [comment]
Alarm state / machine alarm is an error that comes from the machine, lasergrbl only show it.
Screenshot of what messages are shown can help. I suggest to contact ortur in its support forum or facebook group anyway

**Comment by LaserNovice77** [comment]
Thanks for your response.  I’ll reach out to Ortur support and see if they can help me resolve the problem.  Attached is a screenshot of the message when I get the alarm.

**Comment by arkypita** [comment]
using reply-to email doesn't work with attachment, you should go to https://github.com/arkypita/LaserGRBL/discussions/1953 and upload (drag-drop or paste) the screenshot.

**Reply by LaserNovice77** [reply]
Here is a screenshot of the entire screen in LaserGRBL when the alarm occured.
![LaserGRBL Alarm](https://user-images.githubusercontent.com/118078794/201655781-d58d277d-030e-4cd8-9e17-b218d16715f5.png)

**Comment by arkypita** [comment]
I am sorry but from the picture I cannot guess what the problem is. Machine went in alarm state, but no message in console.
Let me know what Ortur say about this issue.

**Comment by LaserNovice77** [comment]
Ortur had me enter a G-code ($RST=*) to restore my engraver to its default settings, power off the machine and then download and install the latest firmware update which resolved the problem.
